app/iht/views/application/asset/household/household_overview.scala.html

```diff
@@ -55,31 +55,31 @@
                     generic_overview_table_heading_item(id = AssetsHouseholdOwnID,
                         questionText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",deceasedName),
                         questionScreenReaderText = Messages("iht.estateReport.assets.householdAndPersonalItemsOwnedByDeceased.title",
```

Comment:
I've fixed the first comment, but I can't see which part of the story I've missed. Could you comment on the message keys that need updating?